bin/replugged.js

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env node
+
+const directory = process.cwd();
+const asar = require("@electron/asar");
+const { Parcel } = require("@parcel/core");
+const {
+  cpSync,
+  rmSync,
+  writeFileSync,
+  copyFileSync,
+  existsSync,
+  mkdirSync,
+  readFileSync,
+} = require("fs");
+const esbuild = require("esbuild");
+const { join } = require("path");
+
+function bundleAddon() {
+  const manifest = JSON.parse(readFileSync("dist/manifest.json", "utf-8"));
```

Comment:
Delete the dist folder here so that maps that are already built won't be included in the bundle